### ISSUE TITLE
SQL inclusion

### ISSUE THREAD

**Issue by rexoctavian** [mobibean/codesnack-ide-feature-requests#90]
Feature request to allow for sql - or just plain text file editing.

**Comment by bushev** [comment]
Hey, do you mean spell checker for SQL?

**Comment by rexoctavian** [comment]
When creating a file SQL is not on the drop down list.

Maybe I am missing something.  Thanks for the reply.

**Comment by bushev** [comment]
Yeah, that will be fixed in the next release for sure! We’ll also add a syntax checker for SQL.
